src/libs/Log.js

```diff
@@ -37,6 +37,7 @@ function LogCommand(parameters) {
 function serverLoggingCallback(logger, params) {
     const requestParams = params;
     requestParams.shouldProcessImmediately = false;
+    requestParams.shouldRetry = false;
```

Comment:
One weird side effect of trigger the option where all the requests should fail is that some iteration of a Log command is getting retried forever. I suspect it's something like:

1. Log fails
2. We run out of retries for a Log command
3. We log that we ran out of retries and log again
4. etc.

But not 100% sure and need to debug some more. I think `Log` is a "persistable" request (but isn't today) so once we have a backoff logic in place we can retry it forever - but at some rate of backoff. For now, we would be making many requests over and over again if we had a situation like "Auth down" so I am making Log not retryable.

Reply:
Yeah, I think we need more debugging to find out that